test/quantization/test_backward_compatibility.py

```diff
@@ -69,6 +71,69 @@ def remove_prefix(text, prefix):
         self.assertEqual(qmodule_scripted(input_tensor), expected, atol=prec)
         self.assertEqual(qmodule_traced(input_tensor), expected, atol=prec)
 
+    def _test_op_graph(self, qmodule, subname=None, input_size=None, input_quantized=True,
```

Comment:
looks like we are copying some code here, is it possible to dedup?

Reply:
sg, reused a bit more code, hope that works!